apps/web/src/components/layout/components/BridgeStatus.tsx

```diff
@@ -1,16 +1,16 @@
 import { Badge, Text } from '@mantine/core';
 import { useQuery } from '@tanstack/react-query';
 import { api } from '../../../api/api.client';
-import { useEnvironment } from '../../../hooks';
+import { useEnvironment } from '../../../hooks/index';
 import { Popover } from '@novu/design-system';
 import { useDisclosure } from '@mantine/hooks';
-import { IS_DOCKER_HOSTED } from '../../../config';
+import { IS_DOCKER_HOSTED } from '../../../config/index';
 
 export function EchoStatus() {
   const [opened, { close, open }] = useDisclosure(false);
 
   const { environment } = useEnvironment();
-  const echoEnabled = !!environment?.echo?.url && !IS_DOCKER_HOSTED;
+  const isBridgeEnabled = !!environment?.echo?.url && !IS_DOCKER_HOSTED;
```

Comment:
We will have a follow-up PR to start setting `environment.bridge.url`, then we'll run a manual migration against MongoDB to backfill the value everywhere, then we will kill `environment.echo.url` for good.